arch/none/optee/src/arch_main.c

```diff
@@ -24,6 +24,21 @@
 #include <scmi_agents.h>
 #include <fwk_core.h>
 #include <mod_optee_mhu.h>
+#include <internal/fwk_context.h>
+
+extern short int thread_get_id(void);
```

Comment:
We have just removed multi-threading and we don't want to add code for a different implementation of multit-threading.

Reply:
In optee case, we have several channels which are fully independent as they don't apply on the same resources and we want to run them in parallel so we have using different event list. thread_get_id() is the optee function to know which channel is running and use the associated linked list. But there is no cross list interaction